doc/source/data/api/dataset.rst

```diff
@@ -153,15 +152,3 @@ Serialization
    Dataset.has_serializable_lineage
    Dataset.serialize_lineage
    Dataset.deserialize_lineage
-
-Internals
----------
-
-.. autosummary::
-   :toctree: doc/
-
-   Dataset.__init__
-   Dataset.dataset_format
-   Dataset.fully_executed
```

Comment:
is materialize documented anywhere?

Reply:
Yeah, it's documented on line 143 under the "Execution" header